tests/e2e/util.go

```diff
@@ -546,6 +546,85 @@ func getPersistentVolumeSpec(fcdID string, persistentVolumeReclaimPolicy v1.Pers
 	return pv
 }
 
+// function to create PVC spec with given namespace, labels and pvName
+func getPersistentVolumeClaimSpecForRMX(namespace string, labels map[string]string, pvName string) *v1.PersistentVolumeClaim {
```

Comment:
Can we use same convention in method name, PVC spec method has RMX and PV spec method has RWX